detections/endpoint/windows_rdp_file_execution.yml

```
@@ -0,0 +1,96 @@
+data_source: []
+name: Windows RDP File Execution
+id: 0b6b12b9-8ba9-48fe-b3b8-b4e3e1cd22b4
+version: 1
+date: '2024-11-21'
+author: Michael Haag, Splunk
+type: TTP
+status: production
+description: The following analytic detects when a Windows RDP client attempts to execute an RDP file from a temporary directory, downloads directory, or Outlook directories. This detection is significant as it can indicate an attempt for an adversary to delivery a .rdp file, which may be leveraged by attackers to control or exfiltrate data. If confirmed malicious, this activity could lead to unauthorized access, data theft, or further lateral movement within the network. 
+search: '| tstats count min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes 
+  where Processes.process IN ("*\\AppData\\Local\\Temp\\*", "*\\AppData\\Local\\Microsoft\\Outlook\\*", "*\\Content.Outlook\\*", "*\\Downloads\\*")
```

Adding the Olk path, as it is used by the new outlook 

```suggestion
  where Processes.process IN ("*\\AppData\\Local\\Temp\\*", "*\\Olk\\Attachments\\*", "*\\AppData\\Local\\Microsoft\\Outlook\\*", "*\\Content.Outlook\\*", "*\\Downloads\\*")
```